templates/overrides/search/facets-result-item.html.twig

```diff
@@ -16,10 +16,13 @@
  * @ingroup themeable
  */
 #}
+{% set attributes = create_attribute() %}
+{% set attributes = attributes.addClass('me-2') %}
```

Comment:
Suggestion, you could do both to simplify as follows:
 set attributes = create_attribute({'class': ['me-2']})